client/src/main/scala/org/apache/celeborn/client/WorkerStatusTracker.scala

```diff
@@ -53,10 +53,18 @@ class WorkerStatusTracker(
     }
   }
 
-  def isWorkerAvailable(worker: WorkerInfo) = {
+  def workerAvailable(worker: WorkerInfo) = {
```

Comment:
The function name `isXXX` is fine, it's consistent with others `verbXYZ`
```
isWorkerAvailable
isWorkerOfPartitionExcluded
```

Reply:
I feel it a bit verbose..

Reply:
done